cmd/terramate/cli/cli.go

```diff
@@ -1667,13 +1682,13 @@ func lookupProject(wd string) (prj project, found bool, err error) {
 
 			logger.Trace().Msg("Load root config.")
 
-			cfg, err := hcl.ParseDir(rootdir, rootdir)
+			cfg, err := config.LoadTree(rootdir, rootdir)
```

Comment:
Thinking if reloadcfg could be called loadcfg and be called here too, it seems to just load the config in general (there is no reload specific behavior I think).

Reply:
new implementation has no need to reload configuration anymore, now the commands that do change configuration also reload the sub portion of the tree that changed.